nsqadmin/options.go

```diff
@@ -43,5 +45,6 @@ func NewOptions() *Options {
 		StatsdInterval:           60 * time.Second,
 		HTTPClientConnectTimeout: 2 * time.Second,
 		HTTPClientRequestTimeout: 5 * time.Second,
+		AllowConfigFromCIDR:      "127.0.0.1/8",
```

Comment:
Agreed, this is what I would have made the default. And the /config API, introduced in a4015c2aba012af68037dccc8c834a5e365daef8, was never in a release ... so you could consider this to not be breaking any previous release.